docs/src/api/class-formdata.md

```diff
@@ -0,0 +1,37 @@
+# class: FormData
+* langs: java
+
+The [FormData] is used create form data that is sent via [APIRequestContext].
+
+```java
+import com.microsoft.playwright.options.FormData;
+...
+FormData form = FormData.create()
+    .set("firstName", "John")
+    .set("lastName", "Doe")
+    .set("age", 30);
+page.request().post("http://localhost/submit", RequestOptions.create().setForm(form));
+```
+
+## method: FormData.create
+- returns: <[FormData]>
+
+Creates new instance of [FormData].
+
+## method: FormData.set
```

Comment:
Can you do the same name multiple times? If so, I'd call this `add`.

Reply:
Same here, we can add `append` if need be.